appsignals-tests/contract-tests/src/test/java/software/amazon/opentelemetry/appsignals/test/jdbc/JdbcContractTestBase.java

```diff
@@ -301,6 +326,15 @@ protected void assertSuccess(
     assertThat(response.status().isSuccess()).isTrue();
 
     var traces = mockCollectorClient.getTraces();
+    //      assertThat(traces)
+    //              .satisfiesOnlyOnce(
+    //                      rss -> {
+    //                          assertThat(rss.getSpan().getKind()).isEqualTo(SPAN_KIND_CLIENT);
+    //                          var attributesList = rss.getSpan().getAttributesList();
+    //                          assertAwsAttributes(
+    //                                  attributesList, method, path, dbSystem, dbOperation, type,
+    // identifier);
+    //                      });
```

Comment:
Why is this commented out?

Reply:
was testing locally forgot to clean before rebasing :(
mybad there will fix it, thanks for pointing out !